e2e_test/source_inline/kafka/include_key_as.slt

```diff
@@ -0,0 +1,284 @@
+control substitution on
```

Comment:
The changes to this test can be reviewed commit by commit

Reply:
what does this command mean?

Reply:
Replace `${}` to env vars in the slt.

To use `${RISEDEV_KAFKA_WITH_OPTIONS_COMMON}`, this needs to be opened.